server/src/test/scala/actors/persistent/nebo/NeboArrivalActorSpec.scala

```diff
@@ -63,20 +66,19 @@ object NeboArrivalActorTest {
           sender() ! state
 
         case _: SaveSnapshotSuccess =>
+          testProbeRef ! "saveSnapshotSuccess"
           ackIfRequired()
 
-        case SaveSnapshot =>
-          testProbeRef ! "snapshot"
-          log.info(s"Received request to snapshot")
-          takeSnapshot(stateToMessage)
+        case SaveSnapshotFailure(md, cause) =>
+          testProbeRef ! "SaveSnapshotFailure"
+          log.error(s"Save snapshot failure: $md", cause)
 
         case m => log.warn(s"Got unexpected message: $m")
       }
 
       override def takeSnapshot(stateToSnapshot: GeneratedMessage): Unit = {
```

Comment:
Again here all you need to do is send a message to the probe after calling the super function rather than repeating all the lines from the super function